src/components/screens/sprint-view-screen.tsx

```diff
@@ -1,11 +1,270 @@
+                {strings.sprint.toDo} 
+              </MenuItem>
+              <MenuItem key={2} value={strings.sprint.inProgress} onClick={() => setFilter("INPROGRESS")}>
+                {strings.sprint.inProgress}
+              </MenuItem>
+              <MenuItem key={3} value={strings.sprint.allTasks} onClick={() => setFilter("DONE")}>
+                {strings.sprint.completed}
+              </MenuItem>
+              <MenuItem key={4} value={strings.sprint.allTasks} onClick={() => setFilter("")}>
+                {strings.sprint.allTasks}
+              </MenuItem>
+            </Select>
+          </FormControl>
+          <Card sx={{ margin: 0, width: "100%", height: "100", marginBottom: "16px" , marginTop: "16px", padding: "0px",
+            '& .negative-value': {
+              color: 'red',
+            }}}>
+            <DataGrid
+              sx={{
+                borderTop: 0,
+                borderLeft: 0,
+                borderRight: 0,
+                borderBottom: 0,
+                '& .header-color': {
+                  backgroundColor: '#f2f2f2',
+                }
+              }} 
+              autoHeight={true}
+              localeText={{ noResultsOverlayLabel: strings.sprint.notFound }}
+              disableColumnFilter
+              hideFooter={true}            
+              rows={allocations}
+              columns={[
+                { 
+                  field: 'projectName',  
+                  headerClassName: 'header-color',                
+                  filterable: false,
+                  headerName: strings.sprint.myAllocation, 
+                  flex: 2, valueGetter: (params) => getProjectName(params.row),
+                  renderCell:(params) => <><Box minWidth="45px" style={{marginRight:"10px"}} component="span" sx={{ bgcolor: getProjectColor(params.row), height: 25, borderRadius: "5px"}} />{getProjectName(params.row)}</>
+                },
+                { 
+                  field: 'allocation', 
+                  headerClassName: 'header-color',
+                  headerName: strings.sprint.allocation, 
+                  flex: 1, valueGetter: (params) => getHoursAndMinutes(totalAllocations(params.row))
+                },
+                { 
+                  field: 'timeEntries', 
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.timeEntries, 
+                  flex: 1, valueGetter: (params) => getHoursAndMinutes(getTotalTimeEntries(params.row) || 0),
+                },
+                { 
+                  field: 'allocationsLeft', 
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.allocationLeft, 
+                  flex: 1, cellClassName: (params) =>  timeLeft(params.row) < 0 ? "negative-value" : "", valueGetter: (params) => getHoursAndMinutes(timeLeft(params.row))
+                },
+              ]}             
```

Comment:
Improve render method code readability by creating a separate columns variable to contain this. Similar was done for vacations in the vacation-requests-table/vacation-requests-table-columns